src/main/java/com/endava/soa_5/Driver.java

```diff
@@ -11,11 +11,11 @@
 
 public class Driver {
     final String PROPERTY_FILE_NAME = "Configuration.properties";
+    PropertyReader propertyReader = new PropertyReader(PROPERTY_FILE_NAME);
```

Comment:
why did you make the propertyReader global?

Reply:
resolved, now isn't global